.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: trigger engine jenkins job
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the addChangesFromVXGI4.19 branch
+  push:
+    branches:
+      - Features/addChangesFromVXGI4.19
+jobs:
+  curl:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: jenkins
+      uses: enflo/jenkins-action@master
+      with:
+        url: "http://jenkins.vr-tual.media:8080"
+        user: "alfred"
+        token: ${{ secrets.JENKINS_TOKEN }}
```

Comment:
Where is that token stored?

Reply:
It's a github secret stored in the settings of this repo.
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets